packages/manager/modules/sign-up/src/form/activity/activity.html

```diff
@@ -159,6 +160,7 @@
             data-on-change="$ctrl.resetCorporationData()"
             data-label="{{:: 'sign_up_activity_corporation_not_created_title' | translate }}"
             data-values="[false]"
+            data-ng-click="$ctrl.trackClick($ctrl.COMPANY_NOT_CREATED_PREFIX)"
```

Comment:
you can use directives `data-track-on`, `data-track-type` and `data-track-name` to avoid creating a specific function in the controller

Reply:
Thanks, I tried to do so but I got an error because it implies that several directives (ouiSelectPicker and trackOn) on the custom component ask for an isolated scope. In this case we have to use the function.